content/pt/docs/concepts/glossary.md

```diff
@@ -0,0 +1,368 @@
+---
+title: Glossário
+description: >-
+  Terminologias utilizadas pelo projeto OpenTelemetry que você pode ou não estar
+  familiarizado.
+weight: 200
+default_lang_commit: 926e23631291ed0ba85563cdaf344a134d0edd8d
+---
+
+O projeto OpenTelemetry usa terminologias que você talvez não conheça. Além
+disso, o projeto também define algumas terminologias de uma maneira diferente
+das outras. Esta página reúne as terminologias utilizadas no projeto e seus
+respectivos significados.
+
+## Terminologia Genérica {#generic-terminology}
+
+### **Agregação** {#aggregation}
+
+O processo de combinar múltiplas medições em estatísticas exatas ou estimadas
+sobre as medições que aconteceram durante um intervalo de tempo ou execução do
+programa. Utilizado pela [Fonte de Dados](#data-source) de uma
+[Métrica](#metric).
+
+### **API**
+
+Interface de Programação de Aplicações (_Application Programming Interface_). No
+projeto OpenTelemetry, é utilizada para definir como os dados de telemetria são
+gerados por [Fonte de Dados](#data-source).
+
+### **Aplicação** {#application}
+
+Um ou mais [Serviços](#service) destinados a usuários finais ou outras
+aplicações.
+
+### **APM**
+
+O Monitoramento de Desempenho de Aplicações (_Application Performance
+Monitoring_) envolve monitorar aplicações de software, sua performance
+(velocidade, confiabilidade, disponibilidade, etc.) para detectar problemas,
+alertar e fornecer ferramentas que permitam encontrar a causa raiz.
+
+### **Atributo** {#attribute}
+
+Termo do OpenTelemetry para [Metadados](#metadata). Adiciona informações
+chave-valor à entidade que está produzindo telemetria. Usado em
+[Sinais](#signal) e [Recursos](#resource). Consulte a [especificação de
+atributos][attribute].
+
+### **Instrumentação automática** {#automatic-instrumentation}
+
+Refere-se a métodos de coleta de telemetria que não exigem que o usuário final
+modifique o código-fonte da aplicação. Os métodos variam conforme a linguagem de
+programação e os exemplos incluem injeção de _bytecode_ ou _monkey patching_.
+
+### **Baggage**
+
+Um mecanismo para propagar [Metadados](#metadata) para ajudar a estabelecer uma
+relação casual entre eventos e serviços. Consulte as [especificações de
+Bagagge][baggage].
+
+### **Biblioteca de cliente** {#client-library}
+
+Veja [Biblioteca instrumentada](#instrumented-library).
+
+### **Client-side app**
```

Comment:
From here on is still all english, so will take another look when it's ready

Reply:
Thanks for the comments @maryliag! Had to stop (_and work_) for a while, so I pushed it as a draft. Anyway, thanks for the review so far!